.github/workflows/core_code_review_weu.yaml

```diff
@@ -1,5 +1,10 @@
 name: WEU - Core TF Validation (PROD)
 
+# This pipeline start automatically when a PR is opened
```

Comment:
# This pipeline starts automatically when a PR is opened.

It is responsible for managing only the legacy part (West Europe), ensuring that changes have occurred exclusively within the directories listed in "paths." It ignores the folders related to the new mode, indicated by the symbol '!', which denotes an excluded path.